src/olympia/addons/cron.py

```diff
@@ -180,27 +180,36 @@ def deliver_hotness(chunk_size=300):
     threshold = 250 if addon type is theme, else 1000
     hotness = (a-b) / b if a > threshold and b > 1 else 0
     """
-    frozen_guids = list(set(fa.addon.guid for fa in FrozenAddon.objects.all()))
+    frozen_guids = list(
+        set(fa.addon.guid for fa in FrozenAddon.objects.all() if fa.addon.guid)
+    )
     log.info('Found %s frozen add-on GUIDs.', len(frozen_guids))
 
-    averages = get_averages_by_addon_from_bigquery(
-        today=date.today(), exclude=frozen_guids
+    amo_guids = (
+        Addon.objects.filter(status__in=amo.REVIEWED_STATUSES)
+        .exclude(guid__in=frozen_guids)
+        .exclude(guid__isnull=True)
+        .exclude(guid__exact='')
+        .exclude(hotness=0)
```

Comment:
The more I think about this, the more I think we should exclude _inactive_ add-ons. They are not visible on any public page. We also include other reasons for an add-on being - temporarily or permanently - being hidden from public pages, by filtering on `REVIEWED_STATUSES`, so why wouldn't we want to exclude _inactive_ ones as well?

Reply:
I think the fewer add-ons we exclude, the better. Excluding based on things the developers can trivially change themselves like user-disabled is risky, because then they might keep an old `hotness` value and suddenly show up as trending later when they are re-enabled if the value they had before being disabled was high.

There is no drawback in resetting hotness for add-ons that are not public, except for the fact that it makes us fetch more in the cron.

Reply:
The same would apply to add-ons that have status _Incomplete_ because there is currently no public version (might have been rejected by a reviewer or disabled by the developer). At any time, the developer could upload a new listed version and in the vast majority of cases, it would be auto-approved and the add-on would become visible again.

Reply:
Yes, and I think we should also not exclude those.

Reply:
@willdurand with that new logic, I don't see a reason to filter by `status` anymore. We do want to continue excluding `hotness=0` to avoid unnecessary updates, excluding empty `guids` because that won't lead to anything, and excluding frozen guids because they should never be updated, but I don't see why we'd continue filtering by status. In any case if there is some data in bigquery those add-ons will get their hotness updated.

Reply:
afaik that was there before? (and it'd be consistent with `average_daily_users`?

Reply:
I removed the `.filter()`